portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,42 +34,44 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<String> comments;
-
   @Override
-  public void init() {
-    comments = new ArrayList<>();
-    comments.add("Sample Comment 1");
-    comments.add("Sample Comment 2");
-    comments.add("Sample Comment 3");
-  }
+  public void init() {}
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(comments);
+    
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    Query query = new Query("Comment").addSort("comment");
+    PreparedQuery results = datastore.prepare(query);
+
+    List<Comment> comments = new ArrayList<>();
+    for(Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      String commentContent = (String) entity.getProperty("comment");
+      System.out.println(commentContent);
```

Comment:
Is for debug purposes?

If so, we don't usually push debug code to the master branch, and if we do, we'll add a comment to explain what's going on with it.